### DISCUSSION TITLE
renaming of files with forbidden symbols

### DISCUSSION THREAD

**Discussion by Aliaksandr3by** [doublecmd/doublecmd#1067] [Suggestions]
### Description

to add possibility edit a wrong file name on renaming;
![image](https://user-images.githubusercontent.com/11432724/208224688-ce57ce71-a27e-40a8-a438-6a1be5f61905.png)

**Comment by avmaksimov** [comment]
If you press Abort or Esc you can edit filename in place - not in the dialog!
